com.github.utsushi.Utsushi.json

```diff
@@ -127,7 +185,11 @@
 				{
 					"type": "git",
 					"url": "https://gitlab.com/utsushi/utsushi.git",
-					"commit": "ab7063358c9819dd37129c6ba58da1009302bdd7"
+					"commit": "839d06a5a80b353cb604eb9f7d352a1648ab1fdf"
```

Comment:
any reason to change the revision?

Reply:
That's the latest commit, the manifest was following a commit  https://gitlab.com/utsushi/utsushi/-/commit/ab7063358c9819dd37129c6ba58da1009302bdd7 which is a year old and there were some translation updates and minor build related corrections.

Reply:
and CI build failures....

Reply:
Yea, I saw that but I haven't noticed those errors when building the Flatpak locally or here. My guess is the CI system change from Devuan Beowulf to Chimaera caused the issues, since it started failing with that commit. 

Anyway I can change it to the last commit that passed in the CI which is https://gitlab.com/utsushi/utsushi/-/commit/85d9379250db0cb3960d283a8886824680d4d314...